src/core/libmaven/alignment/polyFit.cpp

```diff
@@ -1,7 +1,172 @@
-#include "PolyAligner.h"
+#include "mzAligner.h"
```

Comment:
Same thing here. Do not include all the header files here. Seems like a lot of unwanted header files are included here.